ext/date/php_date.stub.php

```diff
@@ -393,13 +393,7 @@ public static function __set_state(array $array) {}
 
 class DatePeriod implements IteratorAggregate
 {
-    /**
-     * @param DateTimeInterface|string $start
-     * @param DateInterval|int $interval
-     * @param DateTimeInterface|int $end
-     * @param int $options
-     */
-    public function __construct($start, $interval = UNKNOWN, $end = UNKNOWN, $options = UNKNOWN) {}
+    public function __construct(DateTimeInterface|string $start, DateInterval|int|null $interval = null, DateTimeInterface|int|null $end = null, int $options = 0) {}
```

Comment:
Should also be reverted? At the very least the defaults are wrong now.

Reply:
Ops, surely!